modules/operator-preconfig-tls-routes.adoc

```diff
@@ -23,18 +23,11 @@ The following table outlines the valid options:
 // | None (Not for production) | Unmanaged | Unmanaged | No | Sets a passthrough route, allows HTTP traffic directly from the route and into the Pod
 |===
 
-
-[NOTE]
```

Comment:
This note is updated in the first file in this PR - should it just be updated here rather than removed?